packages/x-data-grid/src/locales/urPK.ts

```diff
@@ -59,9 +59,9 @@ const urPKGrid: Partial<GridLocaleText> = {
 
   // Filter operators text
   filterOperatorContains: 'شامل ہے',
-  // filterOperatorDoesNotContain: 'does not contain',
+  فilterOperatorDoesNotContain: 'موجود نہیں',
   filterOperatorEquals: 'برابر ہے',
-  // filterOperatorDoesNotEqual: 'does not equal',
+  filterOperatorDoesNotEqual: 'برابر نہیں',
```

Comment:
ٖFollow the convention used in other translation.
```suggestion
  filterOperatorDoesNotEqual: 'برابر نہیں ہے',
```